.github/ISSUE_TEMPLATE/guest-request.yml

```diff
@@ -0,0 +1,75 @@
+name: Guest Request
+description: Complete this form to request to be a guest on eCHO  
+title: eCHO - [PROJECT NAME]  
+labels: ["guest-request", "pending"]  
+assignees: ["lizrice", "xmulligan"]  
```

Comment:
Would @thebsdbox be a better fit here?

Reply:
Is it okay for you @thebsdbox to be assigned to the issues?